Controllers/LoginController.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace MVCCore.Controllers
+{
+    public class LoginController : Controller
+    {
+        private IMediator _mediator;   // Depency Injection to inject Imediator in asp.netcore
+
+        public LoginController(IMediator mediator)
+        {
+            _mediator =  mediator;
+        }
+
+        // GET: LoginController
+        public IActionResult Login(Login login)
+        {
+            LoginHandler handler = new LoginHandler();
```

Comment:
what is the use of dependency injection if we gonna make object like this LoginHandler handler = new LoginHandler();?

Reply:
Modification done , Implemented ILogin